utils/hct/hctdb.py

```diff
@@ -5848,9 +5848,97 @@ def UFI(name, **mappings):
         )
         next_op_idx += 1
 
-        # End of DXIL 1.9 opcodes.
+        self.add_dxil_op(
+              "MatVecMul",
+              next_op_idx,
+              "MatVecMul",
+              "Matrix-Vector Multiply",
+              "<hfwi,<hfwi",
+              "",
+              [
+                  db_dxil_param(0, "$x0", "outputVector", "output vector"),
+                  db_dxil_param(2, "$x1", "inputVector", "input vector"),
+                  db_dxil_param(3, "i1", "isInputSigned", "input signed op kind"),
```

Comment:
```suggestion
                  db_dxil_param(3, "i1", "isInputUnsigned", "input signed op kind"),
```

Reply:
Should I make is `isInputUnsigned" and keep it consistent with the builtin name? And the values `1` and `0` are self explanatory?

Reply:
That sounds good to me - I'll get the proposal updated to match once the dust has settled.

Reply:
Catching up. This has been addressed.

Reply:
Oh, this hasn't been addressed.  It needs to be named `isInputUnsigned` and the description updated to match. Same for the output ones.